docs/aws/audit/elbmonitoring/rules/elb_app_tier_security_policy.mdx

```diff
@@ -23,6 +23,192 @@ AWSWAF
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. First, you need to install and configure AWS CLI on your local machine. You can do this by following the instructions provided by AWS. Make sure you have the necessary permissions to access the Elastic Load Balancer (ELB) resources.
+
+2. Once the AWS CLI is set up, you can list all the load balancers in your account using the following command:
+
+   ```
+   aws elbv2 describe-load-balancers
+   ```
+
+   This command will return a JSON object with details about all your load balancers.
+
+3. To check the SSL policies of each load balancer, you need to describe the listeners for each load balancer. You can do this using the following command:
+
+   ```
+   aws elbv2 describe-listeners --load-balancer-arn <load-balancer-arn>
+   ```
+
+   Replace `<load-balancer-arn>` with the ARN of your load balancer. This command will return a JSON object with details about the listeners for your load balancer.
+
+4. To specifically check the SSL policy, you need to look at the 'SslPolicy' field in the JSON object returned by the previous command. If the 'SslPolicy' field is not set to the latest security policy, then the load balancer is not using the latest SSL negotiations.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can do this by running the command `pip install boto3` in your terminal.
+
+2. Import the necessary modules and establish a session with AWS using your credentials. Here's a basic setup:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='YOUR_REGION'
+)
+```
+
+3. Now, create a client for ELB (Elastic Load Balancer) and retrieve all the load balancers. Then, for each load balancer, retrieve its policies and check if the latest AWS security policy for SSL negotiations is used.
+
+```python
+elb = session.client('elbv2')
+
+load_balancers = elb.describe_load_balancers()
+for lb in load_balancers['LoadBalancers']:
+    lb_arn = lb['LoadBalancerArn']
+    policies = elb.describe_ssl_policies(LoadBalancerArn=lb_arn)
+    for policy in policies['Policies']:
+        if 'ELBSecurityPolicy' not in policy['PolicyName']:
+            print(f"Load balancer {lb['LoadBalancerName']} is not using the latest AWS security policy for SSL negotiations.")
+```
+
+4. The above script will print out the names of all load balancers that are not using the latest AWS security policy for SSL negotiations. If no such load balancers are found, the script will not output anything. This way, you can easily detect any misconfigurations in your AWS Elastic Load Balancers.
+
+Remember to replace `'YOUR_ACCESS_KEY'`, `'YOUR_SECRET_KEY'`, and `'YOUR_REGION'` with your actual AWS access key, secret key, and region, respectively.
+</Accordion>
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>